packages/url/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@rollup/plugin-url",
+  "version": "3.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Import files as data-URIs or to ES6 modules.",
+  "license": "MIT",
+  "repository": "rollup/plugins",
+  "author": "Arpad Borsos <arpad.borsos@googlemail.com>",
+  "homepage": "https://github.com/rollup/plugins/packages/url/#readme",
+  "bugs": "https://github.com/rollup/plugins/issues",
+  "main": "dist/index.js",
+  "engines": {
+    "node": ">=10.0.0"
+  },
+  "scripts": {
+    "build": "rollup -c",
+    "lint": "pnpm run lint:js && pnpm run lint:docs && pnpm run lint:package",
+    "lint:docs": "prettier --single-quote --write README.md",
+    "lint:js": "eslint --fix --cache src test",
+    "lint:package": "prettier --write package.json --plugin=prettier-plugin-package",
+    "prebuild": "rimraf dist/*",
+    "prepublish": "npm test",
+    "pretest": "npm run build",
+    "test": "mocha --require @babel/register"
+  },
+  "files": [
+    "dist",
+    "README.md",
+    "LICENSE"
+  ],
+  "keywords": [
+    "rollup",
+    "plugin",
+    "url"
+  ],
+  "peerDependencies": {
+    "rollup": ">=0.60.0"
+  },
+  "dependencies": {
+    "mime": "^2.4.4",
+    "rollup-pluginutils": "^2.8.2"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.7.2",
+    "@babel/preset-env": "^7.7.1",
+    "@babel/register": "^7.7.0",
+    "mocha": "^6.2.2",
+    "rimraf": "^2.7.1",
+    "rollup": "^1.27.2",
+    "rollup-plugin-babel": "^4.3.3"
+  },
+  "ava": {
+    "files": [
+      "!**/fixtures/**",
+      "!**/helpers/**",
+      "!**/recipes/**",
+      "!**/types.ts"
+    ],
+    "babel": {
+      "extensions": [
+        "png",
+        "svg"
+      ]
+    }
+  },
+  "jsnext:main": "src/index.js",
+  "maintainers": [
```

Comment:
these sections won't be pertinent moving forward as we'll have the `Plugin Maintainers` team on the Rollup org

Reply:
Thanks! Which section(s) other than `maintainers` are you referring to?

Reply:
Probably `authors` and `contributors` too, but I haven't seen those in a plugin yet.